puterbot/visualize.py

```diff
@@ -25,7 +25,7 @@
     rows2dicts,
 )
 
-
+#Global variables & Formatting:
```

Comment:
Block comments should have one space before the pound sign (#) and the comment itself.